### PR TITLE
Cleanup stopping openstack and pacemaker

### PR DESCRIPTION
**Pull request by jsuchome**
Followup for https://github.com/crowbar/crowbar-core/pull/179

### CODE REVIEW

File: chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb [hunk 1]
```diff
@@ -63,17 +63,23 @@
 
 elsif node["crowbar_wall"]["crowbar_db_shutdown"]
 
-  # Now, stop the database.
-  # In case of HA setup, also stop remaining pacemaker resources and HA services.
+  # Stop remaining pacemaker resources
+  bash "stop pacemaker resources" do
+    code <<-EOF
+      for type in clone ms primitive; do
```
Comment (jsuchome) [inline chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb]: Note that I'm not listing `group` here; this is the same as in 4-5 upgrade script where we hadn't group as well.

**Comment by jsuchome** [comment]
@vuntz this should address your comments from previous PR

File: chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb]: Ah, interesting, I thought we were. I guess it works because we stop one item in the group.

**Comment by vuntz** [comment]
Thanks for the cleanups, +1.

**Comment by jsuchome** [comment]
@vuntz are you sure? I didn't address your `g-postgresql` comment...

**Comment by rhafer** [comment]
+1

**Pull request merged**